distributions/README.md

```diff
@@ -2,16 +2,17 @@
 
 Support for a distribution implies all the duties covered by the Approver and Maintainer role requirements. In addition, support means that the Collector SIG is the owner and maintainer of the binaries/images of the different Collector distributions and is responsible for the pipeline that produces those artifacts.
 
+The Collector SIG must support at least one distribution that is recommended for production which includes support for Prometheus, Jaeger, Zipkin, and OpenCensus.
```

Comment:
Jaeger and OpenCensus will be deprecated and removed at some point. Will we update this one later or better rephrase it now?

Reply:
I say we update it once we aren't being held to that expectation.  As far as I am aware we still are. @jpkrohling can you confirm?

Reply:
They will, but they are not yet. I'd keep them here, and remove when time comes.